view-user/src/components/common/Button/Button.tsx

```diff
@@ -1,29 +1,23 @@
+// import React from "react";
+import React, { FC, ReactNode } from "react";
 import styles from "./Button.module.css";
-import classNames from "classnames";
-import { ReactNode } from "react";
 
 interface ButtonProps {
   children: ReactNode;
-  size: string;
-  shape: string;
+  inversion: boolean;
```

Comment:
```suggestion
  inversion?: boolean;
```
オプショナルで大丈夫です。?をつけることで存在しなくてもいい状態にできるのでコンポーネントを呼ぶ際にinversionを書かなくてもエラーになりません。

https://commte.net/nextjs-optional-properties